test/functionalTest/cases/0000_bad_requests/pagination_error_in_uri_params.test

```diff
@@ -93,16 +93,16 @@ Date: REGEX(.*)
 
 
 +++++ 3. URI param 'limit' == 0
-HTTP/1.1 400 Bad Request
-Content-Length: 135
+HTTP/1.1 200 OK
+Content-Length: 86
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
-    "orionError": {
+    "errorCode": {
         "code": "400",
-        "details": "Bad pagination limit: /000000/ [a value of ZERO is unacceptable]",
+        "details": "JSON Parse Error",
```

Comment:
"JSON Parse Error" doesn't seem to be a right message in this case, as actually the request in step 3 doesn't have any payload...

In fact, a OK response seems to be better.

Reply:
> "JSON Parse Error" doesn't seem to be a right message in this case, as actually the request in step 3 doesn't have any payload...
> 
> In fact, a OK response seems to be better.

In step 3 everything is OK but it shows "JSON Parse Error". I didn't find reason for this error. Could you please suggest what modification should be done in this test case?

Reply:
I understand that the response in this case should be the same you get when do a /v1/queryContext operation without any matching result. You can test yourself using this curl:

```
curl -vvv localhost:1026/v1/queryContext -H 'content-type: application/json' -d @- <<EOF
{
    "entities": [
        {
            "type": "foo",
            "isPattern": "false",
            "id": "bar"
        }
    ]
}
EOF
```

You should get something like this:

```
< HTTP/1.1 200 OK
< Connection: Keep-Alive
< Content-Length: 70
< Content-Type: application/json
< Fiware-Correlator: aaee0e26-4621-11ec-9ae3-000c29583ca5
< Date: Mon, 15 Nov 2021 14:38:15 GMT
< 
* Connection #0 to host localhost left intact
{"errorCode":{"code":"404","reasonPhrase":"No context element found"}}
```

The step in the step should resemble that. The reason behind the "JSON Parser Error" in this case should be analyzed.